packages/berry-pnpify/sources/generateSdk.ts

```diff
@@ -2,47 +2,73 @@ import {Filename, NodeFS, PortablePath, xfs, ppath} from '@berry/fslib';
 
 import {dynamicRequire}                             from './dynamicRequire';
 
-const TEMPLATE = (relPnpApiPath: string) => [
+const TEMPLATE = (relPnpApiPath: string, module: string, pnpify: boolean) => [
```

Comment:
Clarity nit (also to avoid the [boolean trap](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap)):
```suggestion
const TEMPLATE = (relPnpApiPath: string, module: string, {usePnpify}: {usePnpify: boolean}) => [
```